compose/install/other.md

```diff
@@ -18,6 +18,9 @@ On this page you can find instructions on how to install the Compose standalone
   $ curl -SL https://github.com/docker/compose/releases/download/{{site.compose_version}}/docker-compose-linux-x86_64 -o /usr/local/bin/docker-compose
   ```
 2. Apply executable permissions to the standalone binary in the target path for the installation.
+  ```console
+  $ sudo chmod +x /usr/local/bin/docker-compose
```

Comment:
Was the `sudo` needed to make it work? (Not at my computer right now to try the steps, but curious if it's needed, or if it works without)

Reply:
Yes, `sudo` is needed because the previous `curl` is downloading the `docker-compose` in the root directory, `/usr/local/bin`. In fact, the `curl` command would also need `sudo` to begin with.